.maestro/shared/sync_verify_bookmarks.yaml

```diff
@@ -3,9 +3,17 @@ appId: com.duckduckgo.mobile.ios
 
 - tapOn: Settings
 - tapOn: Done
+- runFlow:
```

Comment:
The prompt to download missing icons can sometimes appear before the Bookmarks button is tapped, causing the test to fail as it  prevents the Bookmarks from being tapped so added in additional handling for this